src/java/org/apache/cassandra/service/paxos/PaxosPrepare.java

```diff
@@ -877,24 +909,28 @@ public synchronized void onRefreshSuccess(Ballot isSupersededBy, InetAddressAndP
         final DecoratedKey partitionKey;
         final TableMetadata table;
 
-        AbstractRequest(Ballot ballot, Electorate electorate, SinglePartitionReadCommand read, boolean isForWrite)
+        final boolean isForRepair;
```

Comment:
this appears to be unused

Reply:
Hmm I wonder if that was some kind of error because it was actually in use!